lib/src/main/java/com/tbruyelle/rxpermissions2/RxPermissionsFragment.java

```diff
@@ -9,18 +9,17 @@
 import android.support.v4.app.FragmentActivity;
 import android.util.Log;
 
-import java.util.HashMap;
-import java.util.Map;
-
 import io.reactivex.subjects.PublishSubject;
+import java.util.LinkedHashMap;
+import java.util.Map;
 
 public class RxPermissionsFragment extends Fragment {
 
     private static final int PERMISSIONS_REQUEST_CODE = 42;
 
     // Contains all the current permission requests.
     // Once granted or denied, they are removed from it.
-    private Map<String, PublishSubject<Permission>> mSubjects = new HashMap<>();
+    private Map<String, PublishSubject<Permission>> mSubjects = new LinkedHashMap<>();
```

Comment:
What is the point of replacing HashMap with LinkedHashMap? As far as I see, there is no position key access in the code.